### ISSUE TITLE
Docker run fail when using the "use this template" import 

### ISSUE THREAD

**Issue by mgalland** [snakemake-workflows/rna-seq-kallisto-sleuth#11]
Hello, 
I was also making a snakemake workflow (although less evolved) to use Kallisto and Sleuth....

I tried to import the workflow in my own organisation by clicking on the "use this template" button.

All tests passed [see screenshot](https://ibb.co/tX63CqG) except the Docker test.

Do you have an idea why?
Thank you
Marc

```
Run snakemake/snakemake-github-action@v1.4.1
  with:
    directory: .test
    snakefile: workflow/Snakefile
    args: --use-conda --show-failed-logs
/usr/bin/docker run --name af96b420b6d454309b40cba0b188e44ed8cacd_fdba3b --label af96b4 --workdir /github/workspace --rm -e INPUT_DIRECTORY -e INPUT_SNAKEFILE -e INPUT_ARGS -e INPUT_STAGEIN -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/rna-seq-kallisto-sleuth/rna-seq-kallisto-sleuth":"/github/workspace" af96b4:20b6d454309b40cba0b188e44ed8cacd  ".test" "workflow/Snakefile" "--use-conda --show-failed-logs" ""
Building DAG of jobs...
MissingInputException in line 1 of /github/workspace/workflow/rules/trim.smk:
Missing input files for rule cutadapt_pe:
data/reads/a.chr21.2.fq
data/reads/a.chr21.1.fq
##[error]Docker run failed with exit code 1


```

**Comment by dlaehnemann** [comment]
This is most likely fixed once PR #9 is merged. It contains some fixes to faulty file paths that previously caused CI tests to fail when run on workflows created with the "use this template" button. But note that for the tests to pass, you'll also need to checkout the git submodule with the testing data (but instructions for this are included in the new version of the `README.md` in PR #9).

**Comment by dlaehnemann** [comment]
With PR #9 merged, this should be resolved, so I'm closing this issue. Please follow the [new instructions for Testing in the README.md](https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/tree/6d36fb115b7210a5305fa6871ea756aae4d6e04c#testing)
